File: frontend/src/models/Marker/MarkerCreateDTO.ts
import { LngLat } from "@yandex/ymaps3-types";
import Attachment from "../General/Attachment";

interface MarkerCreateDTO {
    name: string,
    placeId: number,
    username: string,
    coordinates: number[],
    attachments: Attachment[],
    description?: string
}

export default MarkerCreateDTO

File: Backend.Application/DTO/Marker/MarkerCreateDTO.cs
using Backend.Domain;

namespace Backend.Application.DTO.Marker
{
    public class MarkerCreateDTO
    {
        public string name { get; set; } = default!;
        public int placeId { get; set; }
        public string username { get; set; } = default!;
        public List<double> coordinates { get; set; } = default!;
        public List<Attachment> attachments { get; set; } = new List<Attachment>();
        //public string[] tags { get; set; } = default!;
        public string? description { get; set; }
    }
}
